Marlin/src/lcd/extui/lib/anycubic_chiron/chiron_tft.cpp

```diff
@@ -292,7 +292,7 @@ namespace Anycubic {
     #if ACDEBUG(AC_SOME)
       serialprintPGM(str);
     #endif
-    while (const char c = pgm_read_byte(str++)) TFTSer.print(c);
+    while (const char c = pgm_read_byte(str++)) TFTSer.write(c);
```

Comment:
I'm almost sure that's the only part that need changes in this patch. `print(char)` prints the value as a number while write output it. `println(const char*)` and `print(const char*)` have not changed behavior